webviews/components/history_view.svelte

```diff
@@ -1,16 +1,175 @@
 <script lang="ts">
-  import type { temporal } from "@temporalio/proto"
-  export let history: temporal.api.history.v1.IHistory
+  import { temporal, google } from "@temporalio/proto"
+
+  export let history: temporal.api.history.v1.History
+  type Timestamp = google.protobuf.ITimestamp
+
+  console.log(history)
+  let duration: number
+  let startTime: number
+  let currentEvent: temporal.api.enums.v1.EventType
+
+  interface WorkflowTask {
+    scheduled?: Date
+    started?: Date
+    startedEventId?: number
+    events: temporal.api.history.v1.IHistoryEvent[]
+    status: "COMPLETED" | "FAILED" | "TIMED_OUT"
+  }
+
+  export function tsToDate(ts: Timestamp | null | undefined): Date {
+    if (ts === undefined || ts === null) {
+      throw new Error(`Expected timestamp, got ${ts}`)
+    }
+    const { seconds, nanos } = ts
+    return new Date((seconds as unknown as number) * 1000 + (nanos || 0) / 1000000)
+  }
+
+  //function to workflowTasks
+  function workflowTasks(history: temporal.api.history.v1.IHistory): WorkflowTask[] {
+    if (history === undefined) {
+      return []
+    }
+
+    let currWFT: WorkflowTask = { events: [], status: "COMPLETED" }
+    let wfts = Array<WorkflowTask>()
+
+    for (const ev of history.events ?? []) {
+      console.log(ev.eventType)
+      switch (ev.eventType) {
+        case temporal.api.enums.v1.EventType.EVENT_TYPE_WORKFLOW_TASK_COMPLETED:
+          wfts.push(currWFT)
+          currWFT = { events: [], status: "COMPLETED" }
+          break
+        case temporal.api.enums.v1.EventType.EVENT_TYPE_WORKFLOW_TASK_FAILED:
+          wfts.push(currWFT)
+          currWFT.events.push(ev)
+          currWFT.status = "FAILED"
```

Comment:
We said we're not showing these: https://github.com/temporalio/vscode-debugger-plugin/issues/6#issuecomment-1207106052